THIRD-PARTY-NOTICE.txt

```diff
@@ -0,0 +1,2501 @@
+Azure SDK for Java uses third-party libraries or other resources that may be
+distributed under licenses different than the Azure SDK for Java software.
+
+In the event that we accidentally failed to list a required notice, please
+bring it to our attention. Post an issue or email us:
+
+           azjavasdkhelp@microsoft.com
+
+The attached notices are provided for information only.
+
+License notice for Azure SDK for Java
+-------------------------------
+
+Copyright (c) Microsoft. All rights reserved.
+Licensed under The MIT License (MIT)
+
+Available at
+https://github.com/Azure/azure-sdk-for-java/blob/master/LICENSE.txt
```

Comment:
You don't need to include the azure-sdk-for-java license in the third-party licenses document.

Reply:
Removed